pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.554</version>
+    <version>1.580</version>
```

Comment:
I bumped this to avoid some strange but not quite investigation worthy test crash. This is just less work.